data/moves.js

```diff
@@ -1294,7 +1294,7 @@ exports.BattleMovedex = {
 				if (move.id === 'skyuppercut' || move.id === 'thunder' || move.id === 'hurricane' || move.id === 'smackdown' || move.id === 'thousandarrows' || move.id === 'helpinghand') {
 					return;
 				}
-				if (source.hasAbility('noguard') || target.hasAbility('noguard')) {
+				if (source.hasAbility('noguard') || target.hasAbility('noguard') || target.volatiles['lockon']) {
```

Comment:
If that's the case, then I should change Lock On to place the volatile on the user of the move and not the target, yes?